SKILL.tsv

```diff
@@ -4724,22 +4724,22 @@ SKILL_20150717_004723	{memo X}Create a Shoggoth molded after the card inserted i
 SKILL_20150717_004724	{memo X}Shoggoth additional attack #{CaptionRatio}#{nl}Shoggoth additonal defense #{CaptionRatio2}#{nl}Use{nl} Plant, Beast, Mutant corpse#{CaptionTime}#
 SKILL_20150717_004725	{memo X}{#DD5500}{ol}[Physidal] - [Pierce]{/}{/}{nl}Install trap that bounces off enemy when stepped on.
 SKILL_20150717_004726	{memo X}Install a Oguon pattern that represents strength noir on the ground. Strength of zombies and allies around the patern increase.
-SKILL_20150717_004727	Physical defense decrease
-SKILL_20150717_004728	Defense decrease
+SKILL_20150717_004727	Decreased physical defense 
+SKILL_20150717_004728	Decreased defense 
 SKILL_20150717_004729	Periodically receive Fire damage
-SKILL_20150717_004730	SP absorption
-SKILL_20150717_004731	Status SP absorbing
-SKILL_20150717_004732	Status by receving SP absorption
+SKILL_20150717_004730	SP Absorption
+SKILL_20150717_004731	SP Absorption Status
+SKILL_20150717_004732	Status from receiving SP Absorption
 SKILL_20150717_004733	Silver increase
 SKILL_20150717_004734	Drop Silver increases
-SKILL_20150717_004735	Magic defense increase
-SKILL_20150717_004736	Magic defense increase, Physical defense increase
+SKILL_20150717_004735	Increase magic defense 
+SKILL_20150717_004736	Decreased magic defense , Increased physical defense 
```

Comment:
Decreased magic defense , Increased physical defense 

to

Decreased magic defense, increased physical defense